### DISCUSSION TITLE
Three new LINQ methods in .NET 9

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#93] [General]
# Three new LINQ methods in .NET 9

Even though we are in the alpha of .NET 9 and .NET 8 was released not more than two months ago, the dotnet team does not sleep and pushes new changes! In this blog post, we are checking what new methods were added to everyones favorite: LINQ.

https://steven-giesel.com/blogPost/0594ba85-356b-47f1-89a9-70e9761c582e

**Comment by wajisurchi** [comment]
Im kurd

**Comment by viceroypenguin** [comment]
FYI: these are available for earlier versions of .net using [SuperLinq](https://github.com/viceroypenguin/SuperLinq).
